### PR TITLE
Fix link to Apache Airflow docs in webserver

### PR DESCRIPTION
**Pull request by turbaszek**
Current link returns 404
https://airflow.apache.org/docs/2.0.0

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: tests/utils/test_docs.py [hunk 1]
```diff
@@ -34,14 +35,20 @@ class TestGetDocsUrl(unittest.TestCase):
...
-            ('1.10.0', 'migration.html', 'https://airflow.apache.org/docs/1.10.0/migration.html'),
         ]
     )
-    def test_should_return_link(self, version, page, expected_urk):
+    def test_should_return_link(self, version, page, expected_url):
         with mock.patch('airflow.version.version', version):
-            self.assertEqual(expected_urk, get_docs_url(page))
+            self.assertEqual(expected_url, get_docs_url(page))
+            resp = requests.get(expected_url)
+            assert resp.status_code == 200
```
Comment (turbaszek) [inline tests/utils/test_docs.py]: Not sure about sending requests (in general it's not best practice) but I don't know other way to make sure what we use works - without this the test gives false positives.

**Review by kaxil — APPROVED** [review]

**Review by TobKed — APPROVED** [review]

**Comment by turbaszek** [comment]
@kaxil should we mark it as 2.0.1?

**Comment by kaxil** [comment]
> @kaxil should we mark it as 2.0.1?

Yup, just marked

File: tests/utils/test_docs.py (hunk 1 above)
Comment (mik-laj) [reply tests/utils/test_docs.py]: It seems fine in this case, but we can speed up this code a bit if we use HEAD requests.

File: tests/utils/test_docs.py [hunk 2]
```diff
@@ -34,14 +35,20 @@ class TestGetDocsUrl(unittest.TestCase):
...
-            ('1.10.0', None, 'https://airflow.apache.org/docs/1.10.0/'),
-            ('1.10.0', 'migration.html', 'https://airflow.apache.org/docs/1.10.0/migration.html'),
         ]
     )
-    def test_should_return_link(self, version, page, expected_urk):
+    def test_should_return_link(self, version, page, expected_url):
         with mock.patch('airflow.version.version', version):
-            self.assertEqual(expected_urk, get_docs_url(page))
+            self.assertEqual(expected_url, get_docs_url(page))
+            resp = requests.get(expected_url)
```
Comment (mik-laj) [inline tests/utils/test_docs.py]:
```suggestion
            resp = requests.head(expected_url)
```

**Review by mik-laj — APPROVED** [review]

**Comment by kaxil** [comment]
Conflicting files here @turbaszek

**Review by kaxil — APPROVED** [review]

**Comment by potiuk** [comment]
The problems were temporary due to wrongly generated constraints (they were there generated from python3.8 in python 3.6 for about an hour by my mistake)

**Pull request merged**